src/wallet/rpcwallet.cpp

```diff
@@ -4106,6 +4062,8 @@ UniValue z_listunspent(const UniValue& params, bool fHelp)
     if (fHelp || params.size() > 4)
         throw runtime_error(
             "z_listunspent ( minconf maxconf includeWatchonly [\"zaddr\",...] )\n"
+            + ShieldedPoolRPCMethodsDeprecationWarning() + "\n"
+            + "Details: shielded pool transactions (t->z, z->z, z->t) " + (AreShieldedPoolRPCMethodsDeprecated() ? "have been " : "are going to be ") + "deprecated.\n"
```

Comment:
I wonder if we should use "disabled" instead of "deprecated", as "deprecation" is typically used with a slightly different meaning and may confuse users.